pay-api/tests/conftest.py

```diff
@@ -114,6 +115,9 @@ def restart_savepoint(sess2, trans):  # pylint: disable=unused-variable
             finally:
                 db.session.remove()
                 transaction.rollback()
+                event.remove(sess, 'after_transaction_end', restart_savepoint)
```

Comment:
Clean up that wasn't happening, it would make other tests fail that weren't using the session fixture.